ballerina/tests/asb_listner_test.bal

```diff
@@ -74,40 +81,43 @@ isolated function increamentTestCaseCount()
     }
 }
 
-@ServiceConfig {
-    queueName: "pre-created-test-queue",
-    peekLockModeEnabled: true,
-    maxConcurrency: 1,
-    prefetchCount: 20,
-    maxAutoLockRenewDuration: 300
-}
-service MessageService on asbListener {
+service on asbListener {
     isolated remote function onMessage(Message message, Caller caller) returns error? {
-        log:printInfo("Message received:" + message.body.toString());
         if message.body == "This is ASB connector test-Message Body".toBytes() {
             if getTestCaseCount() == 0 {
                 setListnerState(RECIEVED);
+            } else if getTestCaseCount() == 1 {
+                var result = caller->complete();
```

Comment:
Any particular reason to use `var` in here?
```suggestion
                var result = caller->complete();
```

Reply:
Fixed with [3a041e72420f43488a2fac41bf69e8f2ae9c6f1f](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/220/commits/3a041e72420f43488a2fac41bf69e8f2ae9c6f1f)